package-lock.json

```diff
@@ -13020,9 +13020,9 @@
       },
       "dependencies": {
         "acorn": {
-          "version": "5.7.3",
-          "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.7.3.tgz",
-          "integrity": "sha512-T/zvzYRfbVojPWahDsE5evJdHb3oJoQfFbsrKM7w5Zcs++Tr257tia3BmMP8XYVjp1S9RZXQMh7gao96BlqZOw==",
+          "version": "5.7.4",
```

Comment:
I ran `npm i` after reinstalling xcode because I got some error when trying to run `npm i` originally.  No diff in the package.json after all that, so this must just be a patch bump by several of our dependencies.  At a glance here are just a couple of packages depending on `acorn`:
```
npx npm-why acorn
npx: installed 4 in 1.718s

  Who required acorn:

  @esri/calcite-components > @storybook/html > @storybook/core > corejs-upgrade-webpack-plugin > webpack > acorn@6.4.1
  @esri/calcite-components > @storybook/html > @storybook/core > webpack > acorn@6.4.1
```